tox.ini

```diff
@@ -14,7 +14,7 @@ setenv =
   QISKIT_SUPRESS_PACKAGING_WARNINGS=Y
   QISKIT_TEST_CAPTURE_STREAMS=1
   QISKIT_PARALLEL=FALSE
-passenv = RAYON_NUM_THREADS OMP_NUM_THREADS QISKIT_PARALLEL RUST_BACKTRACE SETUPTOOLS_ENABLE_FEATURES QISKIT_TESTS QISKIT_IN_PARALLEL
+passenv = RAYON_NUM_THREADS, OMP_NUM_THREADS, QISKIT_PARALLEL, RUST_BACKTRACE, SETUPTOOLS_ENABLE_FEATURES, QISKIT_TESTS, QISKIT_IN_PARALLEL
```

Comment:
There's still whitespace here between the commas and the starts of the next variables.  Did you mean `RAYON_NUM_THREADS,OMP_NUM_THREADS,...`?

Reply:
I thought whitespace + comma is fine like
https://github.com/Qiskit/qiskit-terra/blob/9c1c18cea4b195749a7c9ed7b278f7a32db63b94/tox.ini#L3
Let's see CI's result.

Reply:
CI all passed, so you're almost certainly right.